commander3/todscripts/wmap/huffman.py

```diff
@@ -43,7 +43,7 @@ def __lt__(self, other):
 
 class Huffman:
```

Comment:
This file should be merged with commander3/python/commander_tools/tod_tools/huffman.py